src/shipit_static_analysis/shipit_static_analysis/config.yml

```diff
@@ -0,0 +1,17 @@
+target: obj-x86_64-pc-linux-gnu
+
+# All the clang checks used by the bot
+clang_checkers:
+ - -*
+ - clang-analyzer-deadcode.DeadStores
+ - modernize-loop-convert
+ - modernize-use-default
+ - modernize-raw-string-literal
+ - modernize-use-override
+ - modernize-use-nullptr
```

Comment:
Nit: We purposefully left removed rules commented out in order to leave a trace of why we disabled it (e.g. false positives because of X, controversial because of Y, etc). With this YAML file we lose this information, and might forget why we decided to enable or disable some rules.

Is it possible to re-add commented-out rules with explanations?